wapitiCore/net/crawler.py

```diff
@@ -370,7 +370,7 @@ async def async_try_login(self, auth_url: str, auth_type: str) -> Tuple[bool, di
     async def _async_try_login_basic_digest_ntlm(self, auth_url: str) -> Tuple[bool, dict]:
         page = await self.async_get(web.Request(auth_url))
 
-        if page.status in (401, 404):
+        if page.status in (401, 403, 404):
```

Comment:
I'm not sure if it is relevant to keep the 404 error code here. You can be logged in and receive some 404.

Reply:
I think it is relevant to keep the code 404, because the method `_async_try_login_basic_digest_ntlm` is used by three types of auth:
- basic
- digest
- ntlm
  
Sometimes, the basic auth can be hidden so it can return either 200 or either 404 (https://httpbin.org/#/Auth/get_hidden_basic_auth__user___passwd_).

Reply:
Seems good to me. Rebase please

Reply:
done